articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
-* [ament_lint](https://github.com/ament/ament_lint) contains a set of packages which perform linting tasks.
-  For a set of linters a ROS 2 specific configuration is provided which can be used via the command line as well as from within CMake (commonly as a CTest).
-* [ament_tools](https://github.com/ament/ament_tools) is a Python package which provides command line tools to build, test and install packages.
-  It is similar to [catkin_tools](https://github.com/catkin/catkin_tools) and builds each package in a workspace in topological order.
-  Each different build system is integrated through extension points.
-  While it currently does not build packages in parallel that will be added in the future to speed up the build process.
-* [ament_index](https://github.com/ament/ament_index) contains a set of packages which provide API to access the [ament resource index](https://github.com/ament/ament_cmake/blob/master/ament_cmake_core/doc/resource_index.md).
-  Currently it only contains a Python implementation but other languages like C++ should follow.
+Each package is build separately with its own build system.
+In order to make the output of one package available to other packages each package can extend the environment in a way that downstream packages can find and use its artifacts and resources.
+If the resulting artifacts are installed into e.g. `/usr` it might not be necessary to alter the environment at all since these folders are commonly being searched by various tools.
+
+
+### The ament command line tool
+
+[ament_tools](https://github.com/ament/ament_tools) is a Python package which provides the command line tool `ament` to build, test, install, and uninstall packages.
+It is similar to [catkin_tools](https://github.com/catkin/catkin_tools) and builds each package in a workspace in topological order.
+Support for different build systems is integrated through extension points.
+
+While it currently does not build packages in parallel that feature will be added in the future to speed up the build process.
+The goal is to reuse common functionality from catkin_tools by making it available through a third package which can be used by both tools.
 
 
 ### Integrate arbitrary build systems
 
-Each package can utilize a different build system to perform the steps of configuring, building and installing.
+Each package can utilize a different build system to perform the steps of configuring, building, and installing.
+The build type is defined in each package manifest using the [build_type](http://www.ros.org/reps/rep-0140.html#build-type) tag in the export section.
+
 Currently supported are CMake and Python but support for others (e.g. autotools, plain Makefiles) will likely be added in the future.
-For each of them the native steps are being applied separately by the ament tools.
+For each of them the native steps are being applied separately by the ament tool.
```

Comment:
`them` is not clear here: `For each build system the native steps ...` or `For each package the native steps ...`